src/pytorch_lightning/loggers/tensorboard.py

```diff
@@ -97,13 +107,20 @@ def __init__(
         sub_dir: Optional[_PATH] = None,
         **kwargs: Any,
     ):
+        if not _TENSORBOARD_AVAILABLE:
+            raise ModuleNotFoundError(f"{_TENSORBOARD_AVAILABLE!s}. You can also install `tensorboardX` if you prefer.")
+
```

Comment:
```suggestion
```
this makes TB mandatory not TBX

Reply:
`TensorBoardLogger` can be used with `tensorboard` or `tensorboardX` after this PR. Hence you need to check that at least one of them is installed.

If we support `tensorboard`, it should also work if the user ran `pip install lightning; pip uninstall tensorboardX; pip install tensorboard`